projects/packages/jetpack-mu-wpcom/src/features/wpcom-media/wpcom-export-media-files.php

```diff
@@ -0,0 +1,76 @@
+<?php
+/**
+ * Export Media Files feature for WordPress.com sites.
+ *
+ * @package automattic/jetpack-mu-wpcom
+ */
+
+/**
+ * Registers the Export Media Files admin page.
+ */
+function wpcom_register_export_media_files_page() {
+	add_submenu_page(
+		'tools.php',
+		__( 'Export Media Files', 'jetpack-mu-wpcom' ),
+		__( 'Export Media Files', 'jetpack-mu-wpcom' ),
+		'export',
+		'export-media-files',
+		'wpcom_render_export_media_files_page'
+	);
+}
+add_action( 'admin_menu', 'wpcom_register_export_media_files_page', 8 );
+
+/**
+ * Renders the Export Media Files admin page.
+ */
+function wpcom_render_export_media_files_page() {
+	if ( ! current_user_can( 'export' ) ) {
+		wp_die( esc_html__( 'Sorry, you are not allowed to export content for this site.', 'jetpack-mu-wpcom' ) );
+	}
+
+	// Get the site ID
+	$site_id = get_current_blog_id();
+
+	// Check if this is an Atomic site
+	$is_atomic = ! ( defined( 'IS_WPCOM' ) && IS_WPCOM );
+
+	// For Atomic sites, point to WordPress.com backup instead of wp-admin VaultPress
+	if ( $is_atomic ) {
+		// Get the site domain for the WordPress.com backup URL
+		$site_domain = wp_parse_url( home_url(), PHP_URL_HOST );
+		$backup_url  = 'https://wordpress.com/backup/' . $site_domain;
+		?>
+		<div class="wrap">
+			<h1><?php esc_html_e( 'Export Media Files', 'jetpack-mu-wpcom' ); ?></h1>
+			<p><?php esc_html_e( 'This site is using the more advanced Jetpack VaultPress feature to seamlessly export your files and manage your backups.', 'jetpack-mu-wpcom' ); ?></p>
+			<a href="<?php echo esc_url( $backup_url ); ?>" class="button"><?php esc_html_e( 'Go to Backup', 'jetpack-mu-wpcom' ); ?></a>
+		</div>
+		<?php
+		return;
+	}
+
+	// Get media export URL for Simple sites
```

Comment:
you forgot to `require_once` the php script ;). `$media_export_url` is always false because `function_exists` returns false.
```
require_once WP_CONTENT_DIR . '/lib/wpcom-media-export.php';
```

`function_exists( 'wpcom_get_media_export_url' )` is not necessary because you require the php script and immediately call the function - the function will always exist.

I had another look at the WPCOM code and I believe that `wpcom_site_has_media_files` you've added is not necessary because `wpcom_get_media_export_url` returns `false` if there's nothing to download.

You could replace it with something like:
```
$media_export_url = wpcom_get_media_export_url( $site_id );
if( $media_export_url ) {
  /// Display the html markup
}
```


```